src/inventory_ui.cpp

```diff
@@ -1014,37 +1155,9 @@ item *game::inv_map_for_liquid(const item &liquid, const std::string &title, int
     };
 
     // Buckets can only be filled when on the ground
-    return inv_map_splice( sealable_filter, bucket_filter, sealable_filter, title, radius ).get_item();
-}
-
-int game::inv_for_flag(const std::string &flag, const std::string &title, bool const auto_choose_single)
-{
-    u.inv.restack(&u);
-    u.inv.sort();
-    indexed_invslice reduced_inv = u.inv.slice_filter_by_flag(flag);
-    if (auto_choose_single && reduced_inv.size() == 1) {
-        std::list<item> *cont_stack = reduced_inv[0].first;
-        if (! cont_stack->empty() ) {
-            return reduced_inv[0].second;
-        }
-    }
-    return display_slice(reduced_inv, title);
-}
-
-int game::inv_for_filter(std::string const &title, const item_filter filter)
-{
-    u.inv.restack(&u);
-    u.inv.sort();
-    indexed_invslice reduced_inv = u.inv.slice_filter_by( filter );
-    return display_slice(reduced_inv, title);
-}
-
-int game::inv_for_unequipped(std::string const &title, const item_filter filter)
-{
-    u.inv.restack(&u);
-    u.inv.sort();
-    indexed_invslice reduced_inv = u.inv.slice_filter_by(filter);
-    return display_slice(reduced_inv, title, false);
+    return inv_map_splice( sealable_filter, bucket_filter, sealable_filter, title, radius,
+                           string_format( _( "You don't have a suitable container for carrying %s." ),
+                           liquid.type->nname( 1 ).c_str() ) ).get_item();
```

Comment:
It's probably better to use `item::type_name` instead of accessing the `type` member directly.